src/types/type.ts

```diff
@@ -1,6 +1,58 @@
-export type PrismaModel = 'user' | 'post' | 'member' | 'lesson';
+import {
+  Lesson,
+  LessonBookMark,
+  LessonComment,
+  LessonHashtag,
+  LessonLevel,
+  LessonLevelEvaluation,
+  LessonSolution,
+  MainSkill,
+  Major,
+  Member,
+  MemberFollow,
+  MemberReport,
+  MemberSkill,
+  Post,
+  User,
+} from '@prisma/client';
 
-export type Target = {
-  model?: PrismaModel;
-  field?: string;
+export type PrismaModelName =
+  | 'mainSkill'
+  | 'major'
+  | 'member'
+  | 'memberReport'
+  | 'memberFollow'
+  | 'lesson'
+  | 'lessonLevelEvaluation'
+  | 'lessonSolution'
+  | 'lessonBookMark'
+  | 'lessonLevel'
+  | 'lessonHashtag'
+  | 'lessonComment'
+  | 'memberSkill'
+  | 'post'
+  | 'user';
```

Comment:
```suggestion
export type PrismaModelName = Uncapitalize<Prisma.ModelName>;
```

찾아보니 Prisma에서 ModelName을 대문자로 제공하고 있더라구요!

저희가 만든 IsRecord 데코레이터에선 model name이 파스칼케이스가 아닌 
카멜케이스가 필요하기에 ts에서 제공하는 문자열 도우미 타입인 Uncapitalize을 이용할 수 있겠네요!

Reply:
모델 생길때마다 수동으로 추가해줘야하나 찝찝했는데 좋네요. 남겨주신 방향으로 수정하도록 하겠습니다.